### PR TITLE
Adapting tests to run on T2 chassis by changing rand_one_dut_hostname…

### PR DESCRIPTION
**Pull request by mannytaheri**
### Description of PR
Enhance `test_scp_copy.py` to run on rand_one_dut_hostname(randomly selected dut).  We need the test cases to run on both supervisor and the line card (per hwsku).

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Modified testcases to run on multi-dut.

#### How did you do it?
Replaced 'rand_one_dut_hostname' with 'enum_rand_one_per_hwsku_hostname'.

#### How did you verify/test it?
Ran test cases on a multi duts multi asics chassis.

### CODE REVIEW

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/scp/test_scp_copy.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/platform_tests/test_sequential_restart.py:12:1: F401 'tests.common.fixtures.conn_graph_facts.conn_graph_facts' imported but unused<br/>tests/platform_tests/test_sequential_restart.py:37:1: E302 expected 2 blank lines, found 1<br/>tests/platform_tests/test_sequential_restart.py:49:1: E302 expected 2 blank lines, found 1<br/>tests/platform_tests/test_sequential_restart.py:53:55: E225 missing whitespace around operator<br/>tests/platform_tests/test_sequential_restart.py:69:121: E501 line too long (129 > 120 characters)<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Review by judyjoseph — APPROVED** [review]

**Comment by judyjoseph** [comment]
@mannytaheri could you resolve conflicts. Also can you update the test results in description of PR section.

File: tests/cacl/test_ebtables_application.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def generate_expected_rules(duthost):
     return ebtables_rules
 
 
-def test_ebtables_application(duthosts, enum_rand_one_per_hwsku_hostname, enum_asic_index):
+def test_ebtables_application(request, duthosts, enum_rand_one_per_hwsku_hostname, enum_asic_index):
```
Comment (patrickmacarthur) [inline tests/cacl/test_ebtables_application.py:19]: This change seems at first glance to be unrelated. Is there a reason we need the `request` argument here?

**Comment by mlok-nokia** [comment]
@mannytaheri Please fix the flake8 compliant issue

**Review by abdosi — APPROVED** [review]

**Pull request merged**